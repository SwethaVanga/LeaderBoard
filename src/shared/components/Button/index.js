import React from 'react'
import style from './index.module.scss'

const Button = ({ title, clickAction }) => {
  return (
    <div className={style.button} onClick={clickAction}>
      {title}
    </div>
  )
}

export default Button
