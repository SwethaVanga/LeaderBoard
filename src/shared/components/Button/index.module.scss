@import "../../../assets/style/variables.scss";

.button {
  display: flex;
  justify-content: center;
  align-items: center;
  min-width: 140px;
  min-height: 41px;
  padding: 11px 25px;
  border: 1px solid $header-border;
  border-radius: 6px;
  box-sizing: border-box;
  background: $header;
  outline: none;
  color: $white;

  font-family: Lato;
  font-style: normal;
  font-weight: bold;
  font-size: 12px;
  line-height: 18px;

  cursor: pointer;
}
