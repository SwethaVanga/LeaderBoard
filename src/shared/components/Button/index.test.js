import React from 'react'
import renderer from 'react-test-renderer'
import Button from './index.js'

it('render correctly button component', () => {
  const props = {
    title: '',
    clickAction: () => ('hello'),
  }
  const ButtonComponent = renderer
    .create(<Button title={props.title} clickAction={props.clickAction} />)
    .toJSON()
  expect(ButtonComponent).toMatchSnapshot()
})