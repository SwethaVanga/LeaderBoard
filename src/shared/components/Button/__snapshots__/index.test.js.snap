// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly button component 1`] = `
<div
  className="button"
  onClick={[Function]}
>
  
</div>
`;
